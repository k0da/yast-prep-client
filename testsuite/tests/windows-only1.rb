# encoding: utf-8

# testedfiles: helper1b.yh
module Yast
  class WindowsOnly1Client < Client
    def main
      Yast.include self, "setup-system.rb"

      setup_system("windows-only")

      Yast.include self, "helper1a.rb"

      Yast.import "ProductFeatures"

      ProductFeatures.SetBooleanFeature("partitioning", "try_separate_home", true)
      ProductFeatures.SetBooleanFeature("partitioning", "proposal_lvm", false)
      ProductFeatures.SetBooleanFeature("partitioning", "proposal_snapshots", false)
      ProductFeatures.SetStringFeature("partitioning", "root_max_size", "20 GB")
      ProductFeatures.SetStringFeature("partitioning", "root_base_size", "15 GB")

      Yast.include self, "helper1b.rb"

      nil
    end
  end
end

Yast::WindowsOnly1Client.new.main
