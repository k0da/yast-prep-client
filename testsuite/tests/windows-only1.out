Dump	Proposal:
Dump	Shrink Windows partition /dev/sda1 to 33.21 GiB [destructive]
Dump	Create extended partition /dev/sda2 (41.32 GiB)
Dump	Create swap volume /dev/sda5 (517.72 MiB)
Dump	Create root volume /dev/sda6 (16.50 GiB) with btrfs
Dump	Create volume /dev/sda7 (24.31 GiB) for /home with xfs
Dump	Create subvolume boot/grub2/i386-pc on device /dev/sda6
Dump	Create subvolume opt on device /dev/sda6
Dump	Create subvolume srv on device /dev/sda6
Dump	Create subvolume tmp on device /dev/sda6
Dump	Create subvolume usr/local on device /dev/sda6
Dump	Create subvolume var/crash on device /dev/sda6
Dump	Create subvolume var/lib/mailman on device /dev/sda6
Dump	Create subvolume var/lib/named on device /dev/sda6
Dump	Create subvolume var/lib/pgqsl on device /dev/sda6
Dump	Create subvolume var/log on device /dev/sda6
Dump	Create subvolume var/opt on device /dev/sda6
Dump	Create subvolume var/spool on device /dev/sda6
Dump	Create subvolume var/tmp on device /dev/sda6
Dump	
Dump	Target Map Excerpt:
Dump	device:/dev/sda label:msdos
Dump	device:/dev/sda1 fsid:0x7
Dump	device:/dev/sda2 fsid:0xf
Dump	device:/dev/sda5 fsid:0x82
Dump	device:/dev/sda6 fsid:0x83
Dump	device:/dev/sda7 fsid:0x83
